lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -55,72 +57,68 @@ export class GalaxyCollectionDatasource extends Datasource {
 
     const baseProject = baseUrlResponse.body;
 
-    const versionsUrl = `${baseUrl}versions/`;
+    const versionsUrl = ensureTrailingSlash(joinUrlParts(baseUrl, 'versions'));
 
-    let versionsUrlResponse: HttpResponse<VersionsProjectResult>;
+    let versionsUrlResponse;
     try {
-      versionsUrlResponse = await this.http.getJson<VersionsProjectResult>(
-        versionsUrl
+      versionsUrlResponse = await this.http.getJson(
+        versionsUrl,
+        GalaxyV3Versions
       );
     } catch (err) {
       this.handleGenericErrors(err);
     }
 
     const versionsProject = versionsUrlResponse.body;
 
-    const releases: Release[] = versionsProject.results.map((value) => {
-      const release: Release = {
-        version: value.version,
-        isDeprecated: baseProject.deprecated,
-      };
-      return release;
-    });
+    const releases = await p.map(
```

Comment:
are you sure this improves perf? there is no real parallel processing, main js loop is single threaded

Reply:
This is a left over, when the loop still contained an http request